app/actors/hyku_addons/actors/creator_profile_visibility_actor.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+module HykuAddons
+  module Actors
+    class CreatorProfileVisibilityActor < Hyrax::Actors::AbstractActor
```

Comment:
@BertZZ I thought it would be a good idea to rename this class to better illustrate what it actually related to, rather than a name that was related to the creator field that it edited.